rewrite-java/src/main/java/org/openrewrite/java/LoggingHandler.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java;
+
+@SuppressWarnings("unused")
```

Comment:
?

Reply:
There's a warning on each method about message being unused, because we have no implementation of LoggingHandler. Perhaps I should add a test that includes a LoggingHandler?